test/__snapshots__/github-custom-resource.test.ts.snap

```diff
@@ -39,7 +39,7 @@ Object {
           "S3Bucket": Object {
             "Fn::Sub": "cdk-hnb659fds-assets-\${AWS::AccountId}-\${AWS::Region}",
           },
-          "S3Key": "4babd074a663342a9636c609081fd08a3dfa53a825ee8b6883752c0ddb3ff1cc.zip",
+          "S3Key": "9246bf8aa5fda8c0cd8d04b3bfdfef0d22bea0539f5597ef2b44409ca0717852.zip",
```

Comment:
Thats the reason I dont like complete snapshot tests of whole synthesized stacks :D 
Personally I prefer hasResourceProperties tests but thats probably a discussion like tabs vs spaces in the cdk ecosystem 😝

Reply:
Yeah, and sometimes it gets somehow flaky